.github/workflows/elixir.yml

```diff
@@ -0,0 +1,29 @@
+name: Elixir CI
```

Comment:
This is neat and simple. Shall we also add the build step in this PR itself?

Reply:
It's added to the PR step as well. The `pull_request` key dictates that afaik.